presto-main/src/main/java/com/facebook/presto/sql/planner/plan/PlanNode.java

```diff
@@ -59,6 +59,7 @@
         @JsonSubTypes.Type(value = GroupIdNode.class, name = "groupid"),
         @JsonSubTypes.Type(value = ExplainAnalyzeNode.class, name = "explainAnalyze"),
         @JsonSubTypes.Type(value = ApplyNode.class, name = "apply"),
+        @JsonSubTypes.Type(value = AssignUniqueId.class, name = "uuid"),
```

Comment:
In my first approach I used UUID instead of generated unique id. This is a left over after that approach. Fixed.